src/SprykerEco/Yves/Computop/Mapper/Init/PostPlace/PayPalExpressMapper.php

```diff
@@ -0,0 +1,132 @@
+
+class PayPalExpressMapper extends AbstractMapper
+{
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Generated\Shared\Transfer\ComputopPayPalExpressPaymentTransfer
+     */
+    public function createComputopPaymentTransfer(QuoteTransfer $quoteTransfer)
+    {
+        if ($quoteTransfer->getCustomer() === null) {
+            $quoteTransfer->setCustomer(new CustomerTransfer());
+        }
+        /** @var \Generated\Shared\Transfer\ComputopPayPalExpressPaymentTransfer $computopPaymentTransfer */
+        $computopPaymentTransfer = parent::createComputopPaymentTransfer($quoteTransfer);
+
+        $computopPaymentTransfer->setPayPalMethod($this->config->getPayPalMethod());
+        $computopPaymentTransfer->setMac(
+            $this->computopApiService->generateEncryptedMac(
+                $this->createRequestTransfer($computopPaymentTransfer)
+            )
+        );
+
+        $decryptedValues = $this->computopApiService->getEncryptedArray(
+            $this->getDataSubArray($computopPaymentTransfer),
+            $this->config->getBlowfishPassword()
+        );
+
+        $computopPaymentTransfer->setData($decryptedValues[ComputopApiConfig::DATA]);
+        $computopPaymentTransfer->setLen($decryptedValues[ComputopApiConfig::LENGTH]);
+
+        return $computopPaymentTransfer;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Generated\Shared\Transfer\ComputopPayPalExpressPaymentTransfer
+     */
+    protected function createTransferWithUnencryptedValues(QuoteTransfer $quoteTransfer): ComputopPayPalExpressPaymentTransfer
+    {
+        $computopPaymentTransfer = new ComputopPayPalExpressPaymentTransfer();
+
+        $computopPaymentTransfer->setCapture(
+            $this->getCaptureType(ComputopSharedConfig::PAYMENT_METHOD_PAY_PAL_EXPRESS)
+        );
+        $computopPaymentTransfer->setTransId($this->generateTransId($quoteTransfer));
+        $computopPaymentTransfer->setTxType($this->config->getPayPalTxType());
+
+        $computopPaymentTransfer->setUrlSuccess(
+            $this->router->generate(ComputopRouteProviderPlugin::ROUTE_NAME_PAY_PAL_EXPRESS_PLACE_ORDER, [], Router::ABSOLUTE_URL)
+        );
+        $computopPaymentTransfer->setOrderDesc(
+            $this->computopApiService->getDescriptionValue($quoteTransfer->getItems()->getArrayCopy())
+        );
+
+        $this->mapAddressFromQuoteToComputopPayPalPayment($quoteTransfer, $computopPaymentTransfer);
+
+        return $computopPaymentTransfer;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     * @param \Generated\Shared\Transfer\ComputopPayPalPaymentTransfer $computopPayPalPaymentTransfer
+     *
+     * @return \Generated\Shared\Transfer\ComputopPayPalPaymentTransfer
+     */
+    protected function mapAddressFromQuoteToComputopPayPalPayment(
```

Comment:
Doesn't match to convention for method names
`map[<SourceEntityName>[To<TargetEntityName>]]($sourceEntity, $targetEntity)`